src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const getPageItems = (
+  total: number,
+  itemsPerPage: number,
+  currentPage: number,
+) => {
+  return [...getNumbers(1, total)].slice(
```

Comment:
Consider using a more efficient way to generate the range of numbers for page items. The current implementation with spread operator and slice might not be optimal for large datasets.